### ISSUE TITLE
[Bug] Need to set style.height to remove error text using flutter web

### ISSUE THREAD

**Issue by j-robin-hunter** [AbdulRahmanAlHamali/flutter_typeahead#573]
### Steps to reproduce

Writing a flutter web application and have experienced the following debug output which has been isolated to the inclusion of flutter_typeahead.

```
Width of Platform View type: [_default_document_create_element_invisible] may not be set. Defaulting to `width: 100%`.
Set `style.width` to any appropriate value to stop this message.
```

flutter doctor
```
Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel stable, 3.16.9, on Debian GNU/Linux 11 (bullseye) 6.1.64-09049-g010fe86d9eae, locale en_US.UTF-8)
[✓] Android toolchain - develop for Android devices (Android SDK version 32.0.0-rc1)
[✓] Chrome - develop for the web
[✓] Linux toolchain - develop for Linux desktop
[✓] Android Studio (version 2023.1)
[✓] VS Code (version 1.85.1)
[✓] Connected device (2 available)
[✓] Network resources

• No issues found!

```

pubspec.yaml defines flutter_typeahead: ^5.2.0

very basic TypeAheadField added:

```
TypeAheadField(
  itemBuilder: (BuildContext context, String text) {
    return ListTile(
      title: Text(text),
    );
  },
  onSelected: (_) {},
  suggestionsCallback: (_) {return null;},
),
```

Project has been cleaned and fully rebuilt a number of times. Problem only occurs with TypeAheadField including within the widget. It is a simple StatefulWidget collecting data for a form. All other fields are TextFormFields. Form has a global key

### Expected results

I do not expect this output to occur

### Actual results

Flutter is outputting the text to the debug output

```
Width of Platform View type: [_default_document_create_element_invisible] may not be set. Defaulting to `width: 100%`.
Set `style.width` to any appropriate value to stop this message.
```

### Package Version

5.2.0

### Platform

Web

### Code sample

```
TypeAheadField(
  itemBuilder: (BuildContext context, String text) {
    return ListTile(
      title: Text(text),
    );
  },
  onSelected: (_) {},
  suggestionsCallback: (_) {return null;},
),
```

### Logs

<details open>
<summary>Logs</summary>

```console
[Paste your logs here]
```
</details>

### Screenshots or Video

![Screenshot 2024-03-05 12 06 26](https://github.com/AbdulRahmanAlHamali/flutter_typeahead/assets/35486530/1fb2883f-d239-44e8-8f20-932660571379)

**Comment by clragon** [comment]
The issue you are mentioning appears ot be [this issue](https://github.com/flutter/flutter/issues/141323).
This issue has been fixed in version `0.10.1.+1` of [`pointer_interceptor_web`](https://pub.dev/packages/pointer_interceptor_web) which is a dependency of [`pointer_interceptor`](https://pub.dev/packages/pointer_interceptor).
This package specifies no lock file, so simply upgrading your packages should ensure you use `0.10.1.+1` or higher of `pointer_interceptor_web` which will fix this issue.

**Comment by j-robin-hunter** [comment]
I have added the suggested pointer_interceptor_web: ^0.10.1+1 to the pubspec.yaml file. It has made no difference. Did you check you can reproduce this?
